release/rllib_tests/1gpu_32cpus.yaml

```diff
@@ -1,17 +1,17 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 0
+max_workers: 7
 
 head_node_type:
     name: head_node
-    instance_type: p3.16xlarge
+    instance_type: g3s.xlarge
 
 worker_node_types:
     - name: worker_node
       instance_type: m5.xlarge
-      min_workers: 0
-      max_workers: 0
+      min_workers: 7
+      max_workers: 7
       use_spot: false
```

Comment:
This file is not used.
**BUT** our APPO tests are running on 2 GPU instances, while they could actually only use 1 GPU.
I'm adding this file (and the added 1gpu 32cpus GCE file) in preparation of moving the APPO tests to this cheaper cluster compute. I won't make this part of this PR, which will try to make this migration as low risk as possible, therefore not making any changes to the configs we've been using with AWS.

Reply:
Sound good!

Reply:
APPO needs two, one for tf and one for torch variation. correct?

Reply:
@kouroshHakha The tests are separated, we have rllib_learning_tests_appo_torch and rllib_learning_tests_appo_tf.
Both require 1 GPU but are run as two distinct jobs.